src/operator/cudnn_rnn-inl.h

```diff
@@ -0,0 +1,504 @@
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(cx_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(hy_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(cy_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(dhx_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(dcx_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(dhy_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetTensorNdDescriptor(dcy_desc_,
+                                          dtype_,
+                                          3,
+                                          dimA,
+                                          strideA), CUDNN_STATUS_SUCCESS);
+
+      // Create Dropout descriptors
+      CHECK_EQ(cudnnCreateDropoutDescriptor(&dropout_desc_), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnDropoutGetStatesSize(s->dnn_handle_,
+                                        &dropout_byte_), CUDNN_STATUS_SUCCESS);
+      dropout_size_ = dropout_byte_ / sizeof(DType);
+      CHECK_EQ(cudaMalloc(&dropout_states_, dropout_byte_), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetDropoutDescriptor(dropout_desc_,
+                                        s->dnn_handle_,
+                                        param_.p,  // keep probability
+                                        dropout_states_,
+                                        dropout_byte_,
+                                        seed_), CUDNN_STATUS_SUCCESS);
+      // RNN descriptors
+      CHECK_EQ(cudnnCreateRNNDescriptor(&rnn_desc_), CUDNN_STATUS_SUCCESS);
+      CHECK_EQ(cudnnSetRNNDescriptor(rnn_desc_,
```

Comment:
well...They made some change between cudnn 5.0.4 (April 2016) and cudnn 5.0.5. Previously `cudnnSetRNNDescriptor` has a input parameter `seqLength`. I've updated my cudnn.  It's not a problem anymore.